rules/sinks/third_parties/sdk/intercom/javascript.yaml

```diff
@@ -18,5 +18,5 @@ sinks:
       - "intercom.io"
```

Comment:
intercom domain is intercom.com. they migrated few years back

Reply:
im changing it for all occurrence.